_posts/2023-3-13-collaborative-trailheads.md

```diff
@@ -0,0 +1,75 @@
+
+### Collaborate Trailheads
+{: .flex .flex--left}
+
+The first step in moving this layer over is to move the existing SGID Trailheads data to OSM. Essentially, we need to conflate –or merge– the two datasets to ensure we’re getting the best of both worlds.
+
+To make this process fun (and easy), we created a simple [map challenge](https://maproulette.org/browse/challenges/28673) and opened it up to the mapping community at large. So, if you’ve ever ventured out in Utah, it’s likely you’ll have something to contribute.
+
+In the map challenge, there are essentially two scenarios:
+
+- In one scenario, the SGID trailhead is marked as new to OSM. In that case, you would review the information supplied and either add the missing trailhead to OSM or decline it with a valid reason.
+- In the other scenario, the SGID trailhead is marked as not new to OSM. In this case, you're working on a task to review (and possibly update) existing OSM data. The task is to look at both features and ensure that the most current information is in OSM.
+
+Linked below are three short vidoes that walk you through the map challenge.
+
+### What’s Next for the Trailheads Dataset?
+{: .flex .flex--left}
+
+Once the map challenge is complete, we’ll then direct all maintenance for the trailheads layer to OSM. 
+
+At that point, we will source the SGID Trailheads layer from OSM much like we do with the SGID Open Source Places layer. This involves bringing the OSM data in-house, running a few validations and quality checks, and then serving it through the SGID so it can be consumed as a GIS layer. 
+
+The beauty of this collective stewardship model is that, collectively, we have the tools to keep the data current.
+
+As we move forward with this model, we will also be working with the authoritative stewards to see how this project fits into similar initiatives that are already underway such as the [OSM for Government](https://wiki.openstreetmap.org/wiki/OpenStreetMap_for_Government) and the [Public Domain Map](https://publicdomainmap.org/).
+
+At UGRC, our ultimate goal is to serve relevant, high-quality data that can be used across platforms. By moving certain layers to OSM, we unlock their potential and open them up to the larger mapping community. With increased visibility they become better and more accessible to a variety of users and decision makers.
+
+We hope you’ll join us in this effort.
+
+### Get Involved in the Map Challenge!
+{: .flex .flex--left}
+
+If you’d like to contribute to the project, you can start by watching our three short how-to videos. 
+
+- How to [create an OSM account and link it to the map challenge](https://vimeo.com/807581370)
+- How to [add a new trailhead to OSM](https://vimeo.com/807595086)
```

Comment:
**todo**: you mention a link to the blog post in the show notes that is missing. 

a link to the previous and next videos would be great.

Reply:
good idea

Reply:
done